test/conf/ssh_config_doctor_cluster

```diff
@@ -49,6 +49,15 @@ UserKnownHostsFile=/dev/null
 IdentityAgent=/dev/null
 LogLevel=ERROR
 
+Host localhost
```

Comment:
rename this according to the current naming scheme localhost -> christina.christina